qulice-checkstyle/src/test/java/com/qulice/checkstyle/ChecksTest.java

```diff
@@ -78,6 +78,7 @@
         "ProtectedMethodInFinalClassCheck",
         "NonStaticMethodCheck",
         "ConstantUsageCheck",
+        "ReturnCountCheck",
```

Comment:
@mkordas it would be better to test it in CheckstyleValidatorTest, because it is not our custom rule but one from checkstyle, so we have to check that it exists in our checks.xml